test/e2e/webhook_test.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+	mutatingwebhook "github.com/nmstate/kubernetes-nmstate/pkg/webhook/mutating"
+)
+
+var _ = Describe("Mutating Admission Webhook", func() {
```

Comment:
Please add a comment explaining why we used the label instead of real testing of the set values.